### PR TITLE
add ssl support to mongodb_connect

### PR DESCRIPTION
**Pull request by andre-merzky**
This needs testing before merge.

### CODE REVIEW

**Comment by andre-merzky** [comment]
This has been confirmed to work.

File: src/radical/utils/misc.py [hunk 1]
```diff
@@ -21,12 +21,12 @@ def split_dburl (dburl, default_dburl=None) :
...
     url = ruu.Url (dburl)
 
-    if  not url.schema and not url.host :
+    if not url.schema and not url.host :
         url      = ruu.Url (default_dburl)
         url.path = dburl
 
     # NOTE: add other data base schemes here...
-    if  url.schema not in ['mongodb'] :
+    if url.schema not in ['mongodb', 'ssl+mongodb'] :
```
Comment (marksantcroos) [inline src/radical/utils/misc.py]: `mongodb+ssl`?

File: src/radical/utils/misc.py [hunk 2]
```diff
@@ -21,12 +21,12 @@ def split_dburl (dburl, default_dburl=None) :
...
         raise ValueError ("url must be a 'mongodb://' url, not %s" % dburl)
```
Comment (marksantcroos) [inline src/radical/utils/misc.py]: ... must be a 'mongodb://' or 'mongodb+ssl://'

**Comment by andre-merzky** [comment]
The schema check has been fixed as discussed, and I consider this PR ready for merge review.

**Pull request merged**